src/lib.rs

```diff
@@ -570,7 +586,7 @@ fn start_iter_ids_thread(
                 };
                 obj_tx
                     .send(msg)
-                    .map_err(|e| Error::new(ErrorKind::Other, e.description()))
+                    .map_err(|e| Error::new(ErrorKind::Other, e.to_string()))
```

Comment:
why this change?

Reply:
`desciption` is deprecated according to rustc 1.40.0 -- this fixes the warning.

Reply:
Ah!  Ok.